tools/pepsi/src/power_off.rs

```diff
@@ -1,29 +1,60 @@
 use clap::Args;
 use color_eyre::{eyre::WrapErr, Result};
-
+use constants::HARDWARE_IDS;
+use futures_util::{stream::FuturesUnordered, StreamExt};
 use nao::Nao;
 
-use crate::{parsers::NaoAddress, progress_indicator::ProgressIndicator};
+use crate::{
+    parsers::{number_to_ip, Connection, NaoAddress},
+    progress_indicator::ProgressIndicator,
+};
 
 #[derive(Args)]
 pub struct Arguments {
     /// The NAOs to power off e.g. 20w or 10.1.24.22
-    #[arg(required = true)]
-    pub naos: Vec<NaoAddress>,
+    #[arg(long)]
+    pub naos: Option<Vec<NaoAddress>>,
+
```

Comment:
Please a doc string comment `///` explaining this flag. This will automatically show the correct help text when typing `pepsi poweroff --help`,

Reply:
done